scaffolder-templates/basic-workflow/template.yaml

```diff
@@ -141,10 +117,62 @@ spec:
                   description: Private SSH key associated with the GitHub account - Use Base64 encoding
                   ui:widget: password
                   ui:options: 'rows: 10'
+                persistenceEnabled:
+                  title: Enable Persistance
+                  type: boolean
+                  description: Indicates whether the workflow shall have persistence enabled.
+                  default: false
               required:
                 - namespace
                 - argocdNamespace
                 - sshPrivateKey
+                - persistenceEnabled
+              dependencies:
+                persistenceEnabled:
+                  oneOf:
+                      - properties:
+                          persistenceEnabled:
+                            const: false
+                      - properties:
+                          persistenceEnabled:
+                            const: true
+                          persistencePSQLSecretName:
+                            title: PostgreSQL Secret Name
+                            type: string
+                            default: postgres-secrets
+                            description: Name of the secret in which the PostgreSQL secrets are stored.
```

Comment:
Should we mention here the secret should exist in the same namespace as the workflow?

Reply:
Yes, even for me it's unclear but if it's not in the same NS, there is nothing that will guess the NS in which it should look